comms/core/src/peer_manager/node_identity.rs

```diff
@@ -47,8 +47,12 @@ pub struct NodeIdentity {
     #[serde(deserialize_with = "deserialize_node_id_from_hex")]
     node_id: NodeId,
     public_key: CommsPublicKey,
+    stealth_address_scanning_public_key: CommsPublicKey,
+    stealth_address_spending_public_key: CommsPublicKey,
     features: PeerFeatures,
     secret_key: CommsSecretKey,
+    stealth_address_scanning_secret_key: CommsSecretKey,
+    stealth_address_spending_secret_key: CommsSecretKey,
```

Comment:
Why are we storing both the public and private keys?
Surely we only need to store the private ones?